### ISSUE TITLE
k8s_production/screen.css is not available

### ISSUE THREAD

**Issue by YGeorge** [thoughtworks/build-your-own-radar#297]
CSS is not loaded on the docker image website, I got: 
GET https://www.thoughtworks.com/css/k8s_production/screen.css net::ERR_ABORTED 404

Looks like that may be related to the fix which was made couple of days ago:
https://github.com/thoughtworks/build-your-own-radar/commit/739c1403041883566f16e07bb6cf7f3495a98a42 

@devansh-sharma-tw may you help, please?

**Comment by devansh-sharma-tw** [comment]
Hi @YGeorge ,

To remove any external requests for assets, we integrated the CSS file into the codebase itself (which is the commit you have mentioned).
But to support earlier deployments using non-latest code, we have brought back the link to the CSS file and will keep it alive for the time being. We would recommend updating to the latest image/code for your deployments, if possible.

Now the CSS should be loading as before for your deployment. Please let us know if that is not the case.

Thanks!

**Comment by YGeorge** [comment]
Yes, it works great! Thank you a lot @devansh-sharma-tw for supporting earlier deployments! We will update docker image soon.

Thank you!

**Comment by YGeorge** [comment]
Sorry to disturb you, @devansh-sharma-tw but I noticed it's not available anymore again. Could you please bring it back?
